### PR TITLE
Very basic first implementation to be used as base to complete version 0.1.0

### LINKED ISSUE #9: Add colors for console output
- red for errors 
- green for ok

### PR DESCRIPTION
**Pull request by hrodes**
Basic happy path working.

### CODE REVIEW

File: cmd/enval.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	"os"
+
+	"github.com/fatih/color"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	name = "tchecker"
```
Comment (fllaca) [inline cmd/enval.go]:
```suggestion
	name = "enval"
```

File: cmd/enval.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+)
+
+var cmd = &cobra.Command{
+	Use:  "tchecker",
```
Comment (fllaca) [inline cmd/enval.go]:
```suggestion
	Use:  "enval",
```

File: cmd/enval.go [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+	RunE: executeCmd,
+}
+
+const (
+	manifestFlag = "manifest"
+)
+
+func main() {
+
+	cmd.Flags().String(manifestFlag, "", "path to the manifest file")
```
Comment (fllaca) [inline cmd/enval.go:37]: the default value should be something like `.enval.yaml` or `.enval.manifest.yaml`? Or use the const `DefaultManifestFile` from the config package?

File: cmd/enval.go [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+	} else {
+		manifest, err = config.ReadManifest()
+	}
+
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	toolsStorage := infra.NewDefaultToolsStorage("../../tool-specs")
```
Comment (fllaca) [inline cmd/enval.go:72]: Is this folder a pcker folder? is it ok to be this relative path?

File: pkg/manifestchecker/regexp_version_parser.go [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+	pattern := regexp.MustCompile(parser.regexpPattern)
+
+	match := pattern.FindStringSubmatch(rawVersion)
+	resultMap := make(map[string]string)
+
+	for counter, groupName := range pattern.SubexpNames() {
+		if counter != 0 && parser.isKeyPresent(groupName) {
+			resultMap[groupName] = match[counter]
+		}
+		//if counter != 0 && groupName == "version" {
```
Comment (fllaca) [inline pkg/manifestchecker/regexp_version_parser.go]: remove this commented code

**Review by fllaca — COMMENTED** [review]
It lgtm overall, I made some aesthetical comments. Besides those, I think we can merge this and start iterating on it.

**Review by fllaca — APPROVED** [review]

**Comment by figuerascarlos** [comment]
As starting points great.

**Pull request merged**